src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Updated Semantic Conventions. to v1.21.0. This library can emit either old,
+  new, or both attributes. Users can control which attributes are emitted by
+  setting the environment variable OTEL_SEMCONV_STABILITY_OPT_IN.
+[#4658](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4658)
+
```

Comment:
For consistency, this is the phrasing I've used in the other libraries:

```
* Updated [Semantic Conventions](https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/rpc/rpc-spans.md)
  to v1.21.0. This library can emit either old, new, or both attributes.
  Users can control which attributes are emitted by setting the environment
  variable `OTEL_SEMCONV_STABILITY_OPT_IN`.
  ([#4658](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4658))
```

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4659#discussion_r1264208621